src/components/Dashboard/Overview.vue

```diff
@@ -0,0 +1,24 @@
+<template>
```

Comment:
Please only user upercase names for .vue files. "src/components/**D**ashboard/Overview.vue" to "src/components/**d**ashboard/Overview.vue "